draft-packages/illustration/KaizenDraft/Illustration/Players/VideoPlayer.tsx

```diff
@@ -99,17 +99,15 @@ export const VideoPlayer = ({
     if (prefersReducedMotion) {
       videoElement.pause()
     } else if (autoplay && !prefersReducedMotion) {
-      try {
-        videoElement.play()
-      } catch (e) {
+      videoElement.play().catch(e => {
```

Comment:
[Some older browsers](https://developer.mozilla.org/en-US/docs/Web/API/HTMLMediaElement/play#return_value) don't return promise from `play()`. I know we don't support them so just FYI

Reply:
People using those browsers are going to have way more problems than the empty state not rendering.

Reply:
I know 😉 , I just had ptsd flash from when I was trying to make autoplay work cross browser & iframe & embeds & chinese android forks web views. It took me several weeks to iron out all issues.

Reply:
Careful.. you're going to trigger my ActiveX|Quicktime|Flash PTSD 😜